store/src/java/com/zimbra/cs/mailbox/CalendarItem.java

```diff
@@ -3522,6 +3522,8 @@ private void createPseudoExceptionForSingleInstanceReplyIfNecessary(Invite reply
                         if (itemIdGetter != null) {
                             /* ZWC expects a different mail item id for each exception */
                             localException.setMailItemId(itemIdGetter.get());
+                        } else {
+                            localException.setMailItemId(reply.getMailItemId());
```

Comment:
I would add debug statements here for future troubleshooting.

Reply:
Added Logger, thanks.